lib/src/platform_client.dart

```diff
@@ -585,3 +585,29 @@ class PlatformMessagingKeys {
 
   PlatformMessagingKeys(this.sendKey, this.receiveKey);
 }
+
+/// Get User information from API and return relevant information on the account.
+Future<Session> getUserDetails(String token, int userId) async {
```

Comment:
We don't need to pass in either the `token` or `userId`. The `PlatformClient` has both. We also need to return a different model (`Session` doesn't have what we need).